src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,15 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type TodoLists = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<TodoLists> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map((item) => (
+      <TodoInfo todo={item} key={item.userId} />
```

Comment:
You have `id` in an item, why do you use `userId`?
![image](https://user-images.githubusercontent.com/91411692/231463678-ca782c61-f4cc-4d31-85a8-1be655b7337c.png)
```suggestion
      <TodoInfo todo={item} key={item.id} />
```